From a6a6b94fef886557da07d091787cfdd4b7517a85 Mon Sep 17 00:00:00 2001
From: Michael Weiss <dev.primeos@gmail.com>
Date: Fri, 17 Apr 2020 14:07:12 +0200
Subject: [PATCH] Makefile: Install a symlink named libminigbm.so

This is required for Chromium (only looks for libminigbm.so).
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 35f92f2..b78230e 100644
--- a/Makefile
+++ b/Makefile
@@ -54,6 +54,7 @@ clean: CLEAN($(MINIGBM_FILENAME))
 install: all
 	mkdir -p $(DESTDIR)/$(LIBDIR)
 	install -D -m 755 $(OUT)/$(MINIGBM_FILENAME) $(DESTDIR)/$(LIBDIR)
+	ln -sf $(MINIGBM_FILENAME) $(DESTDIR)/$(LIBDIR)/libminigbm.so
 	ln -sf $(MINIGBM_FILENAME) $(DESTDIR)/$(LIBDIR)/libgbm.so
 	ln -sf $(MINIGBM_FILENAME) $(DESTDIR)/$(LIBDIR)/libgbm.so.$(GBM_VERSION_MAJOR)
 	install -D -m 0644 $(SRC)/gbm.pc $(DESTDIR)$(LIBDIR)/pkgconfig/gbm.pc
-- 
2.25.1

